src/soundbase.h

```diff
@@ -51,6 +51,20 @@ enum EMidiCtlType
     None
 };
 
+enum EPaApiSettings
```

Comment:
Not particularly happy with PortAudio code being in `src/soundbase.h`.  Definitely not happy with WASAPI code being in here.